glean-core/tests/ping.rs

```diff
@@ -10,7 +10,10 @@ use glean_core::CommonMetricData;
 
 #[test]
 fn write_ping_to_disk() {
```

Comment:
Can we file a bug about porting over the tests from `PingMakerTest.kt` (from [glean-ac](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/src/test/java/mozilla/components/service/glean/ping/PingMakerTest.kt)) to Rust? They cover most of the implementation here. Doesn't need to block this PR, it can be done as a follow-up.